src/editor.js

```diff
@@ -264,9 +275,77 @@ const ContentPanel = ({
         codeMirrorElement is a promise resolved by codemirror(), hence the
         until directive. Once resolved, content can be empty.
       -->
+    ${TemplateLightbox({isTemplateLightboxDisplayed})}
     ${until(codeMirrorElement || Textarea({content}))}
   </div>
 `;
+const dispatchLoadHtmlFile = redispatchAs(g('load-html'));
+
+const TemplateLightbox = ({isTemplateLightboxDisplayed}) => html`
+  <div
+    class="${n('template-lightbox')}"
+    ?hidden=${!isTemplateLightboxDisplayed}
+  >
+    ${getNamesOfTemplates()}
+  </div>
+`;
+
+function getNamesOfTemplates() {
+  return html`
+    <div class="${g('flex-center')}">
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="app-install-ads"
+      >
+        <img width="100%" src="/static/app_install.jpg" />
+      </div>
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="multi-image-ads"
+      >
+        <video
+          width="100%"
+          autoplay
+          loop
+          src="/static/multiple-image-template.mp4"
+        ></video>
+      </div>
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="single-video"
+      >
+        <video
+          width="100%"
+          autoplay
+          loop
+          src="/static/single_video.mp4"
+        ></video>
+      </div>
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="single-image-ads"
+      >
+        <video
+          width="100%"
+          autoplay
```

Comment:
omg I opened the deploy preview at full volume and one of these almost gives me a heart attack 😄

Let's add the `muted` attribute to `<video>`s 🙂